test/SpagoSpec.hs

```diff
@@ -645,7 +645,13 @@ spec = around_ setup $ do
       spago ["bundle-app", "--to", "bundle-app.js"] >>= shouldBeSuccess
       checkFixture "bundle-app.js"
 
+    it "Spago should bundle successfully with source map" $ do
 
+      spago ["init"] >>= shouldBeSuccess
+      spago ["bundle-app", "--to", "bundle-app-src-map.js", "--source-maps"] >>= shouldBeSuccess
+      checkFixture "bundle-app-src-map.js"
```

Comment:
We already have a fixture for a `bundle-app` (in the test above), could we use that one instead of adding a new one?